### ISSUE TITLE
The test function generates an InvokeMethodOnNull exception when $this.requiredgroups is not defined

### ISSUE THREAD

**Issue by rdean716** [PowerShell/JEA#11]
```
Configuration Jea_DSC
{
    Import-DscResource -ModuleName JustEnoughAdministration

    JeaEndpoint One
    {
        EndpointName = 'TestEndPoint'
        RoleDefinitions = "@{'NonAdmin_Operator' = @{RoleCapabilities = 'Maintenance'} }"
        TranscriptDirectory = 'C:\Transcripts'
    }

}
```

This sample applies fine to the node. Get-DscConfiguration returns a hashtable as expected with the current configuration; The call to SafeGetValue() by ConvertStringToHashtable in Test-DscConfiguration throws an exception when the RequiredGroup property is not defined.

**Comment by anwather** [comment]
Add pull request with suggested fix

**Comment by anwather** [comment]
@rpsqrd This can be closed now - was fixed in #25

**Comment by rpsqrd** [comment]
Perfect!
